src/index.jsx

```diff
@@ -75,11 +76,11 @@ async function init() {
         <ThemeProvider theme={theme}>
           <MuiThemeProvider>
             <BrowserRouter basename={basename}>
-              <OuterWrapper>
+              <div>
                 { GA.init(gaTracking, dev, gaDebug) && <RouteTracker /> }
                 <ReduxTopBar />
                 <ReduxNavBar />
-                <Box background={background} style={{ width: '100%', margin: 'auto' }}>
```

Comment:
what happens with this `background` prop? should we get rid of the variable?

Reply:
See comments in `src/Login/ProtectedContent.jsx`, I think this `background` variable was not a good design, think we better get rid of it.